chips/earlgrey/src/registers/alert_handler_regs.rs

```diff
@@ -0,0 +1,504 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright lowRISC contributors.
```

Comment:
The Copyright line of the license header should contain a year (as specified in [trd-legal section 4.1](https://github.com/tock/tock/blob/master/doc/reference/trd-legal.md#41-format).

I recommend using one of the following license headers headers for these generated files:

```
// Licensed under the Apache License, Version 2.0 or the MIT License.
// SPDX-License-Identifier: Apache-2.0 OR MIT
// Copyright Tock Contributors 2023.
// Copyright Google LLC 2023.
```

```
// Licensed under the Apache License, Version 2.0 or the MIT License.
// SPDX-License-Identifier: Apache-2.0 OR MIT
// Copyright Tock Contributors 2023.
// Copyright lowRISC Contributors 2023.
```

Reply:
I'm a bit confused by this: the register definitions are an export out of the opentitan codebase.  I don't understand how they acquire "Copyright Tock Contributors".

If it were the case that we're assigning copyright to Tock by contributing the definitions, I can understand then adding that copyright notice.  However, section 3 of the trd-legal document states that "No copyright assignment is required...".

I'm happy to add the notice; I just want to understand why.

Reply:
I think @jrvanwhy's comment was more about the format, i.e. including a year in the copyright statement. I believe the above are just examples of how that could look like. This is to support our CI which automatically checks & enforces that copyright notices are present in files in the Tock codebase.

Indeed, most of the rules of `trd-legal` don't apply to code vendored from other projects (which, I presume, includes these auto-generated code snippets). However, given that the files in this PR do include a copyright notice, and that notice is *very* close to our intended format, it would be nice if they could abide to the following rule from `trd-legal`:

> Copyright notices SHOULD include a year.